cluster-up/cluster/kind/common.sh

```diff
@@ -180,19 +176,12 @@ function setup_kind() {
         docker exec $node /bin/sh -c "curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C /opt/cni/bin"
     done
 
-    echo "ipv6 cni: $IPV6_CNI"
-    if [ -z ${IPV6_CNI+x} ]; then
-        echo "no ipv6, safe to install calico"
-        calico_manifest="$KIND_MANIFESTS_DIR/kube-calico.yaml.in"
-        patched_diff=$(_patch_calico_manifest_diff $calico_manifest)
-        echo "Log Calico manifest diff:"
-        echo "$patched_diff"
-        _patch_calico_manifest "$calico_manifest" "$patched_diff" | _kubectl apply -f -
-    else
-         echo "ipv6 enabled, using kindnet"
-         #currently kind does not fully support ipv6 or ipv6-DualStack,
-         #when using diffrent CNI's.
-    fi
+    echo "Installing Calico"
```

Comment:
I would write "Deploy CNI" since its the place where you deploy the cni plugin and the plugin that we use could vary

Reply:
for now we install calico, i am not sure its important to make it generic, as its specific for the time being,
if you insist i will change

Reply:
We can also add "CNI plugin" or somthing to indicate this is what handels the network and should be done right after the nodes are up, so it will be easier to understand by newcomers, what you think?

Reply:
yep i also wondered if to add the word CNI, and then thought its might be redundant because calico here is known as the CNI, i will add those words.

Reply:
Done

Reply:
Thanks!